db/db.js

```diff
@@ -365,6 +383,17 @@ async function isOverdoseSuspected(xethru, session, location) {
     let condition2 = 1 * (session.still_counter > still_threshold); //seconds
     let condition3 = 1 * (sessionDuration > sessionDuration_threshold);
     let condition4 = 1 * (0);
+
+    let alertReason='NotSet'
```

Comment:
Is it possible for both `condition2` and either `condition1` or `condition3` to be `1` at the same time?

If so, this logic looks like `alertReason` will be set to `Stillness` when maybe you'd expect it to be `Duration and Stillness`. Are you concerned about this level of granularity?

Also, I suggest that you make these strings into constants.

Reply:
My guess would be that we'd want to be more conservative and default to the more severe one, which is stillness?

@sampathsatti what's your take on this?